Bottom: 0384d48c913897b7d3af1ddb5455ef5b7f824bc1
Top:    aafcad086ae3168b264fbcfed756d93078ab185d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-19 16:03:16 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index ed0efdc..b83e45b 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -120,23 +120,13 @@ hidgl_add_triangle (triangle_buffer *buffer,
 
 void hidgl_draw_grid (BoxType *drawn_area);
 void hidgl_set_depth (float depth);
-<<<<<<< current
 void hidgl_draw_line (int cap, Coord width, Coord x1, Coord y1, Coord x2, Coord y2, double scale);
 void hidgl_draw_arc (Coord width, Coord vx, Coord vy, Coord vrx, Coord vry, Angle start_angle, Angle delta_angle, double scale);
 void hidgl_draw_rect (Coord x1, Coord y1, Coord x2, Coord y2);
-void hidgl_fill_circle (Coord vx, Coord vy, Coord vr, double scale);
+void hidgl_fill_circle (Coord vx, Coord vy, Coord vr);
 void hidgl_fill_polygon (int n_coords, Coord *x, Coord *y);
-void hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box, double scale);
-void hidgl_fill_rect (Coord x1, Coord y1, Coord x2, Coord y2);
-=======
-void hidgl_draw_line (int cap, double width, int x1, int y1, int x2, int y2, double scale);
-void hidgl_draw_arc (double width, int vx, int vy, int vrx, int vry, int start_angle, int delta_angle, double scale);
-void hidgl_draw_rect (int x1, int y1, int x2, int y2);
-void hidgl_fill_circle (int vx, int vy, int vr);
-void hidgl_fill_polygon (int n_coords, int *x, int *y);
 void hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box);
-void hidgl_fill_rect (int x1, int y1, int x2, int y2);
->>>>>>> patched
+void hidgl_fill_rect (Coord x1, Coord y1, Coord x2, Coord y2);
 
 void hidgl_init (void);
 int hidgl_stencil_bits (void);
